src/components/sidebar/connection/ConnectionListItem.vue

```diff
@@ -101,6 +110,14 @@ export default {
     },
     duplicate() {
       this.$emit('duplicate', this.config)
+    },
+    copyUrl() {
+      this.close()
+      this.$copyText(this.config.fullConnectionString)
```

Comment:
Should this include the password if it is saved?

Reply:
No, the same getter will be used for the `title` attributes of the connection list item.